packages/linter/src/generators/utils/eslint-file.ts

```diff
@@ -1,12 +1,20 @@
 import type { Tree } from '@nrwl/devkit';
 
-const eslintFileList = ['.eslintrc.json', '.eslintrc.js'];
+export const eslintConfigFileWhitelist = [
+  '.eslintrc',
+  '.eslintrc.js',
+  '.eslintrc.cjs',
+  '.eslintrc.yaml',
+  '.eslintrc.yml',
+  '.eslintrc.json',
+];
 
 export function findEslintFile(tree: Tree): string | null {
-  for (const file of eslintFileList) {
+  for (const file of eslintConfigFileWhitelist) {
```

Comment:
Why did you rename this constant?

Reply:
Hey @meeroslav

I renamed the constant mainly because I thought it would be a more accurate description of what was as I didn't want to leave anything up to assumption, since I found when I was working on this PR, I often found myself wondering if this variable name was too ambiguous.

I not precious about it though, I can revert this change if you like. Please let me know and I will update it for you 👍 

Thanks for reviewing my PR 🥰

Reply:
Makes sense. I would probably go with `eslintConfigFileNames` but I guess this is fine as well.

Reply:
Oh that's an even better suggestion! Maybe you can update it in your next PR, otherwise I can also raise a new PR.